src/main/java/com/github/slem1/await/HttpConnectionConfig.java

```diff
@@ -15,6 +15,8 @@
 
     private int priority = Integer.MAX_VALUE;
 
+    private boolean trustAllCerts;
```

Comment:
I think skipSSLVerification or skipSSLCertVerification be more meaningful for the end user (as it is the parameter in the xml configuration)

Reply:
Yes, sure!